src/components/Dropdown/Dropdown.tsx

```diff
@@ -0,0 +1,128 @@
+import React, {
+  FC,
+  CSSProperties,
+  useState,
+  Children,
+  ReactElement,
+  cloneElement,
+  ReactNode,
+} from 'react';
+import { classNames } from '../../utils/classNames';
+import DropdownOption from './DropdownOption';
+
+export interface IDropdownProps {
+  /** children must be React Element */
+  children?: ReactElement | ReactElement[];
+  /** set customized css class */
+  className?: string;
+  /** set dropDown to be disabled */
+  disabled?: boolean;
+  /** set customized css style */
+  cssStyle?: CSSProperties;
+  /** set default string on active option */
+  placeholder?: string;
+  /** a callback to provide current value */
+  onChange?: (val: string) => void;
```

Comment:
could this val be something else? or just string?

Reply:
onChange will pass value back to parent, and currently value is a string.

Reply:
You already have the logic here, just change the string to be any here and somewhere related and then you can put anything to the value of Options.